package.json

```diff
@@ -34,7 +34,8 @@
     "storybook": "start-storybook -p 6006",
     "test": "npx jest",
     "test:integration": "./scripts/test-integration",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "dev": "tsc-watch -p tsconfig.dist.json"
```

Comment:
this is very similar to `dist`. Should we rname `dist:watch`?

Reply:
I would match it with the build script name.
So in this case I would agree with Russ and name it `dist:watch` - that way it's easier to remember.

We should keep in mind that if we change svgs or update our design tokens we need to execute some additional steps (`npm run build:design-tokens && npm run build:SVG`) but I think that is okay and we should not run it everytime a watch event appears as this would take much longer to build.